index.html

```diff
@@ -271,43 +271,17 @@
       <section id="obj-methods">
         <h2>Methods on objects</h2>
         <p>
-          There are several common operations upon social objects, which are
-          expressed using standard HTTP methods applied to the Object's
-          dereferencable URI.
-          To avoid conflict with existing standards (and hence support
-          "multiprotocol" server implemnetations), the scope of requests which
-          servers are required to honor as defined in this specification is
-          limited as follows:
+          There is a HTTP GET method an object's dereferencable URI that
```

Comment:
This is good.  But I find "There is a HTTP GET method" to be awkward... "of course there's an HTTP GET method" I think, so I think maybe that sentence should be refactored a bit further.  How about:

'The HTTP GET method may be dereferenced against an object's "id" property to retrieve the activity.'

(I think saying "id" explicitly instead of "uri" is more clear too, right?  Esp because there's a uri field and that's for human readable stuff.)

Otherwise, LGTM!